src/main/java/com/buinak/imageapi/service/ImageService.java

```diff
@@ -35,15 +39,23 @@ public Image addImage(String name, String description, MultipartFile file) {
         return imageRepository.saveAndFlush(image);
     }
 
+    public List<String> listImageUrls() {
+        final String BASE_URL =
```

Comment:
make this a static constant field in the service

Reply:
this looks like an expensive operation, but will never change the output in the lifecycle of the application

Reply:
doing this for some reason breaks everything, will skip for now